pkg/service/catalog/api/types.go

```diff
@@ -0,0 +1,21 @@
+package api
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+type CatalogService struct {
+	// Target is a reference to a service
+	Target kapi.ObjectReference
+	// Type is a published service's claim type
+	Type string
+	// Description is a user-defined description of the published service
+	Description string
```

Comment:
> The description for the service in the namespace is "something I exported
for my team" and the description in the catalog is a multi-paragraph intro
to the service including usage info.  Seems pretty different.  Exported
services are targeted at consumers.  Services in the UI are targeted at
namespace owners.

How do you envision being able to set the description on the catalog entry? 2 different annotations on the original service?